src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -466,6 +469,71 @@ RequestContext<object> requestContext
             await requestContext.SendResult(result.First().ToString());
         }
 
+        private async Task HandleGetAllCommandsRequest(
+            object param,
+            RequestContext<object> requestContext)
+        {
+            var psCommand = new PSCommand();
+            psCommand.AddScript("Get-Command -CommandType Function, Cmdlet, ExternalScript | Select-Object Name,ModuleName | Sort-Object Name");
+            Logger.Write(LogLevel.Verbose, $"Calling {psCommand.Commands}");
+            Stopwatch stopwatch = new Stopwatch();
+            stopwatch.Start();
+            var result = await this.editorSession.PowerShellContext.ExecuteCommand<PSObject>(psCommand);
```

Comment:
Given that we expect a type that isn't `PSObject` here, it might be clearer to put `IEnumerable<PSObject>` instead of `var`